java/ql/src/semmle/code/java/frameworks/JavaxJson.qll

```diff
@@ -0,0 +1,62 @@
+/**
+ * Provides models for the `javax.json` package.
+ */
+
+import java
+private import semmle.code.java.dataflow.ExternalFlow
+
+private class FlowSummaries extends SummaryModelCsv {
+  override predicate row(string row) {
+    row =
+      [
+        "javax.json;JsonArray;false;getBoolean;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonArray;false;getBoolean;;;Argument[1];ReturnValue;value",
+        "javax.json;JsonArray;false;getInt;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonArray;false;getInt;;;Argument[1];ReturnValue;value",
+        "javax.json;JsonArray;false;getJsonArray;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonArray;false;getJsonNumber;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonArray;false;getJsonObject;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonArray;false;getJsonString;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonArray;false;getString;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonArray;false;getString;;;Argument[1];ReturnValue;value",
+        "javax.json;JsonArray;false;getValuesAs;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonArrayBuilder;false;add;;;Argument[-1];ReturnValue;value",
+        "javax.json;JsonArrayBuilder;false;add;;;Argument[0];Argument[-1];taint",
+        "javax.json;JsonArrayBuilder;false;addNull;;;Argument[-1];ReturnValue;value",
+        "javax.json;JsonArrayBuilder;false;build;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonNumber;false;bigDecimalValue;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonNumber;false;bigIntegerValue;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonNumber;false;bigIntegerValueExact;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonNumber;false;doubleValue;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonNumber;false;intValue;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonNumber;false;intValueExact;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonNumber;false;longValue;;;Argument[-1];ReturnValue;taint",
+        "javax.json;JsonNumber;false;longValueExact;;;Argument[-1];ReturnValue;taint",
```

Comment:
There is also [`numberValue()`](https://jakarta.ee/specifications/platform/9.1/apidocs/jakarta/json/jsonnumber#numberValue--); that one might be value flow?